rules/library.bzl

```diff
@@ -992,7 +992,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
         lib_names.append(swift_libname)
 
         if objc_sources or cpp_sources:
-            # Add generated swift header to header maps for angle bracket imports
+            # Add generated swift header to header maps for double quote imports
             swift_doublequote_hmap_name = name + "_swift_doublequote_hmap"
```

Comment:
IME we might be able to drop this; given if `-I` includes the same headers. In other-words the comment `double quote` might not be on point if a `-I` include would allow you to quote import a header? Having "doublequote hmap"'s never added up to me

Reply:
Totally agree. I don't think we need those separate "double quote" imports as well. I can delete them in a separate PR.

Reply:
Cool - thanks again for the PR, I ended up testing this out a bit this afternoon after I saw it and pushed it here: https://github.com/bazel-ios/rules_ios/pull/689!